test/src/org/labkey/test/tests/cds/CDSTooltipVerificationTest.java

```diff
@@ -0,0 +1,131 @@
+package org.labkey.test.tests.cds;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.labkey.test.Locator;
+import org.labkey.test.categories.Git;
+import org.labkey.test.pages.cds.LearnGrid;
+import org.labkey.test.util.cds.CDSHelper;
+import org.openqa.selenium.WebElement;
+
+import java.util.List;
+
+@Category({Git.class})
+public class CDSTooltipVerificationTest extends CDSReadOnlyTest
```

Comment:
It isn't necessary to add CDS tests to the `Git` suite. We exclude them on TeamCity.
```suggestion
@Category({})
public class CDSTooltipVerificationTest extends CDSReadOnlyTest
```